src/config/dark-sites.config

```diff
@@ -1,3 +1,4 @@
+^roosterteeth.com
```

Comment:
We actually don't allow allow `^` `$` as of this moment they aren't alphabetic ordered.